Blockzilla/AutocompleteCustomUrlViewController.swift

```diff
@@ -171,7 +171,7 @@ extension AutocompleteCustomUrlViewController: UITableViewDataSource {
         if editingStyle == .delete {
             tableView.beginUpdates()
             _ = customAutocompleteSource.remove(at: indexPath.row)
-            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.customDomainRemoved, object: TelemetryEventObject.setting)
+            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.customDomainRemoved, object: TelemetryEventObject.customDomain)
```

Comment:
If this is a new probe, it's fine to change now since we haven't released it yet. @boek is this OK with you?